packages/workflow/src/records/handler/certify.test.ts

```diff
@@ -55,6 +57,28 @@ describe('Certify record endpoint', () => {
       )
     )
 
+    // Gets the location hierarchy to find the regLastOffice / regLastLocation
+    mswServer.use(
+      rest.get(
+        'http://localhost:2021/locations/ce73938d-a188-4a78-9d19-35dfd4ca6957/hierarchy',
+        (_, res, ctx) => {
+          return res(
+            ctx.json([
+              fixtures.savedAdministrativeLocation({
+                id: 'ce73938d-a188-4a78-9d19-35dfd4ca6957' as UUID
+              }),
+              fixtures.savedLocation({
+                id: '0f7684aa-8c65-4901-8318-bf1e22c247cb' as UUID,
+                partOf: {
+                  reference: 'Location/ce73938d-a188-4a78-9d19-35dfd4ca6957'
+                }
+              })
+            ])
+          )
+        }
+      )
+    )
```

Comment:
we can potentially replace the default location handlers here: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/workflow/test/handlers.ts#L40-L57 with the one you defined

Reply:
👍🏻 I think would be good to address the tests in a separate PR when we've finished changes for metrics/performance as well.